composer.json

```diff
@@ -9,6 +9,8 @@
     "require": {
         "php": ">=5.5",
         "ext-curl": "*",
+        "guzzlehttp/guzzle": "~4.0",
```

Comment:
Yes, but the progress subscriber won't work on guzzle 6. That's why I suggested guzzle 5.

Both Guzzle 5 and Guzzle 6 will be maintained for the same amount of time anyway.

Guzzle 4 is officially dead btw.